<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PyMafia</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 25, 25)

</string>
  </property>
  <widget class="QLabel" name="PyMafiaLogo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Asset/PyMafia_Logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Server_Address">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>배달의민족 도현</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: none;
border-radius: 5px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Server Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Nickname">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>배달의민족 도현</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: none;
border-radius: 5px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Nickname</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Server_Port">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>배달의민족 도현</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: none;
border-radius: 5px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Server Port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EnterButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>500</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>배달의민족 도현</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: none;
border-radius: 3px;</string>
   </property>
   <property name="text">
    <string>방 입장</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
